index.js

```diff
@@ -75,9 +82,11 @@ request(options, function (error, response) {
   let hasError = hasStoreSearchError;
 
   const inventory = Object.entries(skuCounter)
-    .map(([key, value]) => `${SKUS[key]}: ${value}`)
-    .join(" | ");
+    .map(([key, value]) => `${skuList[key]}: ${value}`)
+    .join("\n");
```

Comment:
does adding a new line here create some risk that the text will be cut off when the macOS notification appears? i don't know how tall those things can get.

Reply:
Good question! I'm testing under Windows 10 so I'm not sure. I've updated the code so that the original pipe separator is used in the notification, but the line break is used in console output. Best of both worlds ? :)

Reply:
yup, sounds good to me!